Misc/NEWS.d/next/Core and Builtins/2019-11-20-09-50-58.bpo-37986.o0lmA7.rst

```diff
@@ -0,0 +1,4 @@
+Improve performance of :c:func:`PyLong_FromDouble` for values that fit into
+:c:type:`long`. Now :meth:`float.__trunc__` is faster up to 10%,
+:func:`math.floor()` and :func:`math.ceil()` are faster up to 30% when used
```

Comment:
Are these numbers still correct? There are other changes related to trunc/floor/ceil in 3.9, they can reduce or increase the relative effect of this optimization.

Reply:
I think we could just drop the second sentence and keep the first here.